ccd-definition/CaseEventToFields/CareSupervision/createOrder-nonprod.json

```diff
@@ -0,0 +1,30 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "createOrder",
+    "CaseFieldID": "interimEndDate",
```

Comment:
The one in `prod.json` has a mid event attached to it, wouldn't that mid event need to be in this file as well?

Reply:
I take it back, just seen that it will be merged with the date of issue validation